debian/changelog

```diff
@@ -0,0 +1,5 @@
+litecli (1.10.1) unstable; urgency=medium
+
+  * First packaged release
+
+ -- Joshua M. Boniface <joshua@boniface.me>  Wed, 01 May 2024 04:33:18 -0400
```

Comment:
I put myself as the maintainer here and used the current stable version, but this is of course changeable and should be set to the maintainer at the next release.

Reply:
The changelog would also need to be updated/added to (top of the file so latest is first) for each new release that is built.